src/main/java/calculator/Token.java

```diff
@@ -0,0 +1,52 @@
+package calculator;
+
+import calculator.operation.Addition;
+import calculator.operation.Division;
+import calculator.operation.Multiplication;
+import calculator.operation.Operation;
+import calculator.operation.Subtraction;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Token {
+
+  public static final String OPERAND_PATTERN = "[0-9]+";
+  public static final String OPERATOR_PATTERN = "[+\\-*/]";
+  private static final Map<String, Operation> OPERATIONS_MAP = new HashMap<>();
```

Comment:
변수명에 자료구조명은 없어도 될 것 같아요
```suggestion
  private static final Map<String, Operation> OPERATIONS = new HashMap<>();
```